src/components/header/header.less

```diff
@@ -1,5 +1,24 @@
 
 .header {
-  padding: 2rem;
   background-color: white;
-}
\ No newline at end of file
+  display: flex;
```

Comment:
hvis vi skal bruke grid istedenfor table for overskrift så kan vi bruke "
  display: grid;
  grid-template-columns: auto auto auto;"
og "
display: flex;
flex-direction: column;" kan fjernes

Reply:
Se nytt forslag

Reply:
hvis du skal bruke eksisterende <Filters> komponenten som jeg foreslo da må du bruke følgende css property i header.less:-

display: grid;
grid-template-columns: 400px auto;